src/ansys/dpf/core/operators/serialization/hdf5dpf_generate_result_file.py

```diff
@@ -446,12 +450,14 @@ def time_freq_support_out(self):
     def ansys_unit_system_id(self):
         """Allows to connect ansys_unit_system_id input to the operator.
 
-        Defines the unitsystem the results are
-        exported with.
+        Defines the unit system the results are
+        exported with. a result info can be
+        input to also export physics type and
+        analysis type.
```

Comment:
```suggestion
        exported with. A result info can be
        input to also export physics type and
        analysis type.
```
Also, we should not use the `ansys_unit_system_id` pin to define whether we export an `analysis_type` or a `physics_type`. This makes no sense.